### DISCUSSION TITLE
Guess 🇺🇸 states

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#954] [Galaxy Trivia] [answered]
```
C O N E A L H I W J I F E R S O R O N I E Y A
```

**Comment by peterbaccaro** [comment, marked as answer]
The hidden US states are:

- **CALIFORNIA**
- **OHIO**
- **NEW JERSEY**

![image](https://github.com/ParanoidUser/yolo/assets/74666079/e2d66b4b-a2b0-4226-ac64-c0ee66237708)

**Comment by YashNuhash** [comment]
> Question: Guess US States: "C O N E A L H I W J I F E R S O R O N I E Y A"

## The hidden US states are: 

- CON - Connecticut
- EAL - Illinois
- H - Hawaii
- IWI - Wyoming
- J - Jersey
- IFE - Florida
- R - Rhode Island
- SOR - South Carolina
- ONI - Ohio
- EYA - New York
- Al - Alaska or Alabama
